PaleTree/FrmMain.cs

```diff
@@ -656,13 +656,16 @@ private void OnQueueTimer(object state, EventArgs args)
 				LstPackets.BeginUpdate();
 				foreach (var palePacket in newPackets)
 				{
+                    Boolean isFilter;
```

Comment:
You might want to change your IDE to use tabs, that way there are less changes in the files if a tab-user with an auto-formater comes around.